src/ext/tags.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2016 Resonance Labs, Inc
```

Comment:
actually, we don't have license stuff at the top of the other files. IMO we can just remove this entirely, there's a `LICENSE` in the repo, isn't that sufficient?